src/mumble/GlobalShortcut_win.cpp

```diff
@@ -550,7 +550,20 @@ void GlobalShortcutWin::timeTicked() {
 			default:
 				break;
 		}
-		id->pDID->Poll();
+
+		{
+			QElapsedTimer timer;
+			timer.start();
+
+			id->pDID->Poll();
+
+			// If a call to Poll takes more than
+			// a second, warn the user that they
+			// might have a misbehaving device.
+			if (timer.elapsed() > 1000) {
```

Comment:
Could use ```hasExpired``` (no I don't expect you to change this ;) Just found it in the docs and thought that they have this as a function was mildly interesting ;) )